packages/contract/src/contract-factory.ts

```diff
@@ -92,8 +92,8 @@ export default class ContractFactory {
   createTransactionRequest(deployContractOptions?: DeployContractOptions) {
     const storageSlots = deployContractOptions?.storageSlots
       ?.map(({ key, value }) => ({
-        key: hexlifyWithPrefix(key, true),
-        value: hexlifyWithPrefix(value, true),
+        key: hexlifyWithPrefix(key),
+        value: hexlifyWithPrefix(value),
```

Comment:
Added by mistake? Although that whole function is redundant, we can just use `hexlify`.